vector/src/main/java/im/vector/app/features/home/room/detail/timeline/TimelineEventController.kt

```diff
@@ -415,7 +416,12 @@ class TimelineEventController @Inject constructor(private val dateFormatter: Vec
                         partialState = partialState,
                         lastSentEventIdWithoutReadReceipts = lastSentEventWithoutReadReceipts,
                         callback = callback,
-                        eventsGroup = timelineEventsGroup
+                        eventsGroup = timelineEventsGroup,
+                        reactionsSummaryEvents = ReactionsSummaryEvents(
+                                onAddMoreClicked = { reactionListFactory.onAddMoreClicked(callback, event) },
```

Comment:
I would avoid creating a new data class to be pass everywhere. Would prefer having  ReactionsSummaryCallback as interface with empty default implementation. Then let TimelineEventController.Callback inherits this interface and just delegate to reactionListFactory in the case of onShowLess and onShowMore

Reply:
This was my first attempt. Let me check if I can enhance it fast for the release.